include/solver/helper.h

```diff
@@ -12,8 +12,7 @@
 #include <functional>
 #include <string>
 
-namespace solver {
-namespace helper {
+namespace solver::helper {
```

Comment:
I'm not sure if this is a good practice, but if you do, please also update the closing comments to L109 (`// namespace solver::helper`

Reply:
also in the corresponding helper.cpp, L80

Reply:
The change is suggested by clang-tidy. I just updated the closing comments.